PolyEngine/Engine/Res/Shaders/prefilterCubemap.frag.glsl

```diff
@@ -0,0 +1,102 @@
+#version 330 core
+
+in vec3 vPositionInModel;
+
+uniform samplerCube uEnvCubemap;
+uniform float uRoughness;
+
+out vec4 color;
+
+const float PI = 3.14159265359;
+
+float RadicalInverse_VdC(uint bits)
+{
+    bits = (bits << 16u) | (bits >> 16u);
+    bits = ((bits & 0x55555555u) << 1u) | ((bits & 0xAAAAAAAAu) >> 1u);
+    bits = ((bits & 0x33333333u) << 2u) | ((bits & 0xCCCCCCCCu) >> 2u);
+    bits = ((bits & 0x0F0F0F0Fu) << 4u) | ((bits & 0xF0F0F0F0u) >> 4u);
+    bits = ((bits & 0x00FF00FFu) << 8u) | ((bits & 0xFF00FF00u) >> 8u);
+    return float(bits) * 2.3283064365386963e-10; // / 0x100000000
```

Comment:
As you can se we reuse a lot of code, this further assures me in need of shader includes support.